main.go

```diff
@@ -23,6 +24,7 @@ func init() {
 	flag.String("port", "9680", "The port number to listen on for HTTP requests")
 	flag.String("address", "0.0.0.0", "The address to listen on for HTTP requests")
 	flag.String("log-level", "info", "The minimum logging level; levels are, in ascending order: debug, info, warn, error")
+	flag.String("url", "", "The URL of the SAPControl web service")
```

Comment:
minor: perhaps we should call it `soapUrl` or `wdslUrl` because we never know if we have others urls

Reply:
Hmm, okay, what about `sap-control-url`? SAPControl is the exact name of the web service provided by SAP Start Service.